store/pipeline.go

```diff
@@ -323,3 +323,46 @@ func (s *BoltStore) PipelineDelete(id int) error {
 		return b.Delete(itob(id))
 	})
 }
+
+// SettingsPut puts settings into the store.
+func (s *BoltStore) SettingsPut(c *gaia.StoreConfig) error {
```

Comment:
We should put that into it's own file e.g. store/settings.go

Reply:
Absolutely. I'm just playing around here seeing how it would work.